File: easy/217./217.cs
public class Solution
{
  public bool ContainsDuplicate(int[] nums)
  {
    Dictionary<int, int> countNumber = new Dictionary<int, int>();

    foreach (int num in nums)
    {
      if (countNumber.ContainsKey(num))
      {
        countNumber[num] += 1;
      }
      else
      {
        countNumber.Add(num, 1);
      }
    }

    foreach (KeyValuePair<int, int> entry in countNumber)
    {
      if (entry.Value > 1)
      {
        return true;
      }
    }

    return false;
  }
}

File: easy/217./217.py
class Solution:
 def containsDuplicate(self, nums: List[int]) -> bool:
  dic = dict()

  for i in nums:
   if i in dic:
    dic[i] += 1
   else:
    dic[i] = 1
  
  for v in dic.values():
   if v > 1:
    return True

  return False